packages/internal/src/build/api.ts

```diff
@@ -74,6 +67,7 @@ export const prebuildFile = (
   const code = fs.readFileSync(srcPath, 'utf-8')
   const result = transform(code, {
     cwd: getPaths().api.base,
+    babelrc: false,
```

Comment:
The documentation is so weird for this option I had trouble understanding it. Can you explain what this does? And where you saw it make a difference?

Reply:
Sure! So by default babel will search for babelrc files, event if you specify `configFile`.... unless you specify this option as false. 

The effect this has is - say the user has a project they're upgrading to the latest version, and they didn't remove `api/.babelrc` - the false option forces babel to ignore it.

Reply:
Awesome, so unintuitive, can you add a comment saying something about this? "Even though we specify the config file, babel will still search for it and merge them because we have specified the filename property".

https://babeljs.io/docs/en/options#babelrc

Reply:
Like Siri says... You goddit!

Reply:
Done